db/seeds.rb

```diff
@@ -1,140 +1,187 @@
-                                     geolocation: '(52.455421,0.043325)',
-                                     gias_data: { 'ReligiousCharacter (name)': 'Roman Catholic' },
-                                     name: 'Burnsfield Infant School',
-                                     phase: :primary,
-                                     postcode: 'PE16 6ET',
-                                     region: 'South East England',
-                                     school_type: 'Community School',
-                                     town: 'Chatteris',
-                                     urn: 110628)
-
-leadership = Leadership.limit(1).sample(1).first
+school_1 = FactoryBot.create(:school,
+                             name: "Bexleyheath Academy",
+                             urn: "137138",
+                             phase: :secondary,
+                             url: "http://www.bexleyheathacademy.org/",
+                             minimum_age: 11,
+                             maximum_age: 18,
+                             address: "Woolwich Road",
+                             town: "Bexleyheath",
+                             county: "Kent",
+                             postcode: "DA6 7DA",
+                             region: "London",
+                             easting: "549194",
+                             northing: "175388",
+                             geolocation: '(51.4578146490981,0.146065490118642)',
+                             readable_phases: ["secondary"],
+                             detailed_school_type: "Academy sponsor led",
+                             school_type: "Academies",
+                             gias_data: { 'ReligiousCharacter (name)': 'None' },
+                             school_type: 'Academy')
+
+school_2 = FactoryBot.create(:school,
+                             name: "Upton Cross ACE Academy",
+                             urn: "144523",
+                             phase: :primary,
+                             url: "http://www.uptoncross.kernowlearning.co.uk",
+                             minimum_age: 4,
+                             maximum_age: 11,
+                             address: "Upton Cross",
+                             town: "Liskeard",
+                             county: "Cornwall",
+                             postcode: "PL14 5AX",
+                             region: "London",
+                             easting: "228041",
+                             northing: "72116",
+                             geolocation: '(50.52350433336815,-4.427269703777728)',
+                             readable_phases: ["primary"],
+                             detailed_school_type: "Academy converter",
+                             school_type: "Academies",
+                             gias_data: { 'ReligiousCharacter (name)': 'None' },
+                             school_type: 'Academy')
+
+trust_1 = FactoryBot.create(:school_group,
```

Comment:
can we use the `:trust` factory? Also can we seed a local_authority too?

Reply:
sure